totalopenstation/formats/landxml.py

```diff
@@ -0,0 +1,360 @@
+#! /usr/bin/env python
+# -*- coding: utf-8 -*-
+# filename: landxml.py
+# Copyright 2015 Damien Gaignon <damien.gaignon@gmail.com>
+#
+# This file is part of Total Open Station.
+#
+# Total Open Station is free software: you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Total Open Station is distributed in the hope that it will be
+# useful, but WITHOUT ANY WARRANTY; without even the implied warranty
+# of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Total Open Station.  If not, see
+# <http://www.gnu.org/licenses/>.
+
+import xml.etree.ElementTree as xml
+import time
+import re
+import os
+
+from . import Parser
+
+# Template file relativ to this file path
+TEMPLATE = "../../data/template.xml"
```

Comment:
This path specification will fail very easily when calling the program from outside the source tree. See my comment about putting the file inside the Python source as a string.
Otherwise the right way is to use `pkg_resources`.